packages/wells/src/client/api/wellsApi.ts

```diff
@@ -56,6 +56,7 @@ export class WellsAPI extends ConfigureAPI {
       block: well.block,
       field: well.field,
       operator: well.operator,
+      spudDate: well.spudDate != undefined ? new Date(well.spudDate) : null,
```

Comment:
This is a hack I had to do to be able to call methods on the date object i.e 'toLocalDateString()'. Wasn't able to do that without explicitly creating a object using `new`